ios/RNSScreenStackHeaderConfig.m

```diff
@@ -602,7 +617,8 @@ + (NSMutableDictionary *)blurEffectsForIOSVersion
       @"prominent": @(UIBlurEffectStyleProminent),
     }];
   }
-#if defined(__IPHONE_13_0) && TARGET_OS_IOS
+#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && defined(__IPHONE_13_0) && \
```

Comment:
After attempting to use this branch directly as suggested in this comment: https://github.com/software-mansion/react-native-screens/issues/686#issuecomment-733722458 I was still having issues trying to build to tvOS.

I made this change and it seemed to work.

```suggestion
#if !TARGET_OS_TV && defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && defined(__IPHONE_13_0) && \
```

Reply:
Thank you for checking. Are there any other places where there is an error?

Reply:
Can you check on branch from #675? I added some more checks there because of new features and updated the old ones. If it works OK there, I think we can close this PR in favor of that one.

Reply:
Hey @WoLewicki - Tested #675 on my project and I did not get any errors. 👏